code/modules/reagents/chemistry/holder.dm

```diff
@@ -934,6 +934,8 @@
 						matching_container = TRUE
 				if (isliving(cached_my_atom) && !reaction.mob_react) //Makes it so certain chemical reactions don't occur in mobs
 					matching_container = FALSE
+				if (istype(cached_my_atom, /obj/item/food/grown) && !reaction.plant_react)
+					matching_container = FALSE
```

Comment:
can't you just use mob_react instead of plant_react here? that would also catch almost every other reaction that produces an item, like the plasma solidification reaction.

Reply:
but plants arent a subtype of mobs?